packages/api/src/graphql/blocks.ts

```diff
@@ -540,8 +553,15 @@ export const GraphQLLinkPageBreakBlockInput = new GraphQLInputObjectType({
   name: 'LinkPageBreakBlockInput',
   fields: {
     text: {type: GraphQLString},
+    richText: {type: GraphQLNonNull(GraphQLRichText)},
     linkURL: {type: GraphQLString},
-    linkText: {type: GraphQLString}
+    linkText: {type: GraphQLString},
+    linkTarget: {type: GraphQLString},
+    hideButton: {type: GraphQLBoolean},
```

Comment:
i would make this also `GraphQLNonNull`

Reply:
saa

Reply:
Yes. That's why. A boolean should never be null.